etc/vector/vector.d/90-output.toml.tmpl

```diff
@@ -72,57 +88,34 @@ if is_object(.parsed) {
   del(.parsed)
 }
 
-.logprops = compact(merge(data, extra))
+. = compact(merge(data, extra))
 '''
 
-{{if eq "logfmt" (getenv "LOGS_OUTPUT_FORMAT") }}
-[transforms.all_formatted]
-type = "remap"
-inputs = ["all"]
-source = '''
-logfmt_order=["ts", "app", "chan", "lvl", "msg", "container_name"]
-if is_array(.logfmt_order) {
-  logfmt_order = unique(append(logfmt_order, .logfmt_order) ?? logfmt_order)
-}
-.orig_message = .message
-.message = encode_logfmt(.logprops, logfmt_order) ?? .orig_message
-'''
-{{else}}
-[transforms.all_formatted]
-type = "remap"
-inputs = ["all"]
-source = '''
-
-.orig_message = .message
-.message = encode_json(.logprops)
-'''
-{{end}}
-
 {{if eq "dir" (getenv "LOGS_EXPORT_TARGET") }}
   {{if eq (getenv "LOGS_EXPORT_DIR") "/dev/null" }}
     # LOGS_EXPORT_DIR=/dev/null
     # So we wont enable any sinks.out_file
   {{else}}
     [sinks.out_file]
     type = "file"
-    inputs = ["all_formatted"]
-    path = "{{getenv "LOGS_EXPORT_DIR"}}/{{ getenv "LOGS_EXPORT_FILENAME" "{{.app}}/{{.chan}}.log" }}"
-    encoding.codec = "raw_message"
+    inputs = ["all"]
+    path = "{{getenv "LOGS_EXPORT_DIR"}}/{{ getenv "LOGS_EXPORT_FILENAME" "{{.log_group}}.log" }}"
+    encoding.codec = "json"
     encoding.timestamp_format = "rfc3339"
   {{end}}
 {{else if eq "cloudwatch" (getenv "LOGS_EXPORT_TARGET") }}
 [sinks.out_cloudwatch_logs]
 type = "aws_cloudwatch_logs"
-inputs = ["all_formatted"]
+inputs = ["all"]
 group_name = "{{ getenv "LOGS_EXPORT_CLOUDWATCH_GROUP_NAME" "deskpro" }}"
 stream_name = "{{ getenv "LOGS_EXPORT_CLOUDWATCH_STREAM_NAME" "{{.app}}/{{.chan}}/{{.container_name}}" }}"
```

Comment:
@MattGrundy I left this the same - but do you think the name here should also change?

Reply:
We currently set the group name when we use this but for what we need the stream name currently works fairly nicely, i.e. we're currently happy with the default